DirectX11Proj/ModelData.h

```diff
@@ -26,6 +26,6 @@ struct MeshData
 	std::vector<VertexData> vertices;
 	std::vector<uint32_t> indices;
 
-	std::string textureFilePath;
+	std::string textureFilePath = "empty";
```

Comment:
Since it's a struct I'm leaving it at this initialization, but the reason I chose empty is because it is easy to identify if a texture is empty or not. I will consider "" and have to reconsider it from a debugging perspective.